test/built-ins/Temporal/PlainDateTime/prototype/withPlainTime/argument-object.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
```

Comment:
This test should be named the same as the other similar tests for other methods.

Reply:
done